### DISCUSSION TITLE
Updating identical image used in multiple containers

### DISCUSSION THREAD

**Discussion by EddieFAF** [containrrr/watchtower#1112] [Q&A]
I have two containers using the same image, eg. using nginx for two different website containers. When watchtower is updating the image, the first container gets restarted just fine. The second gets restarted aswell, but gets an ID instaed of the containername defined in the compose file. I also get an error message when udating the first container, that the image couldn't get deleted because it is (ofcourse) still being used by the second container.
Is there any way to tell watchtower that those two containers are related, imagewise and both have to be restartet after one image is updated?

**Comment by simonaronsson** [comment]
If I remember this correctly, the issues described should be solved by #1103 and #622.
